src/components/CityListItem.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="city-item"> 
+        {{ cityToString(city) }}
+        <button @click="$emit('on-edit', city)" :disabled="isEditMode">Edit</button>
+        <button @click="deleteCity(city.id)" :disabled="isEditMode">Delete</button>
+    </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
```

Comment:
When you import something from the same source, you can use: import { mapGetters, mapActions } from 'smoething';